source/daplink/drag-n-drop/flash_manager.c

```diff
@@ -236,6 +239,7 @@ error_t flash_manager_uninit(void)
 
 void flash_manager_set_page_erase(bool enabled)
 {
+    config_ram_set_page_erase(enabled);
```

Comment:
set_erase_method and accept two methods (enums) - sector erase or chip erase wouldn't be better?

Reply:
If I understand you correctly, you mean the following changes:
void config_ram_set_page_erase(bool) ==> void config_ram_set_page_erase(enum)
void flash_manager_set_page_erase(bool) ==> void flash_manager_set_erase(enum)
?